js/functionaldeps.js

```diff
@@ -1,969 +1,5 @@
-/**
- * This class is the container for Functional Dependencies.
- * @param lhs left hand side of the functional dependencies, should be an array of strings
- * @param rhs right hand side of the functional dependencies, should be an array of strings
- * @constructor
- */
-function FunctionalDependency(lhs, rhs){
```

Comment:
Is there a reason you moved this out of this file. (It'll be easier for me to review if the changes are small and obvious).

Reply:
I moved all functions to a file called calculate-lib.js for testing purposes, I have changed two methods, which are _renderFunctionalClosure_ and _renderFunctionalClosure_. I make them return the HTML code instead of directly modifying the UI.